File: totosuki/ABC/B/149.py
A, B, K = map(int, input().split())

if A >= K:
  A -= K
  K = 0
elif A < K:
  K -= A
  A = 0
  if B >= K:
    B -= K
  elif B < K:
    B = 0

print(A, B)

File: OCH_Tao/ABC/B/149.js
let [A, B, K] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ").map(x => parseInt(x, 10));
const X = Math.min(A, K);
A -= X;
K -= X;
const Y = Math.min(B, K);
B -= Y;
K -= Y;
console.log(`${A} ${B}`);